Pipfile

```diff
@@ -0,0 +1,23 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+lxml = "~=3.3.0"
+codalib = "==1.0.3"
+sqlparse = "*"
+mysql-python = "*"
+Django = "~=1.8"
+
+[dev-packages]
+django-debug-toolbar = "*"
+tox = "*"
+pytest = "==3.5.1"
+pytest-django = "==2.9.1"
+mock = "==1.3.0"
+factory-boy = "==2.5.2"
+"flake8" = "*"
```

Comment:
Does this mean something different having `"flake8"` in double quotes here?

Reply:
`pipenv install` updates the Pipfile automatically and this is what it does for this package. I don't know why.